R/estimate_infections.R

```diff
@@ -324,13 +324,12 @@ estimate_infections <- function(reported_cases,
 
   # Initialise fitting by using a previous fit or fitting to cumulative cases
   if (!is.null(args$init_fit)) {
-    if (!inherits(args$init_fit, "stanfit")) {
-      if (args$init_fit %in% "cumulative") {
-        args$init_fit <- init_cumulative_fit(args,
-          warmup = 50, samples = 50,
-          id = id, verbose = FALSE
-        )
-      }
+    if (!inherits(args$init_fit, "stanfit") &&
```

Comment:
just flagging that this and other style changes below should be in their own PR.

Reply:
I'd normally agree but this was in response to a failing `lintr` check on this PR - without adding it it wouldn't have got the green check mark.

Reply:
I agree.